src/script/conversation/ConversationRepository.ts

```diff
@@ -2589,8 +2588,7 @@ export class ConversationRepository {
     }
 
     const isBackendEvent = eventData.otr_archived_ref || eventData.otr_muted_ref;
-    const selfConversation = this.conversationState.selfConversation();
-    const inSelfConversation = selfConversation && matchQualifiedIds(selfConversation, conversationId);
+    const inSelfConversation = this.conversationState.isSelfConversation(conversationId);
```

Comment:
Can we rename it? its a little confusing between inSelfConversation and isSelfConversation

Reply:
We want to know if the message received is, actually, _in_ the self conversation (thus the name of the variable). 
What would you suggest as another name?

Reply:
sounds good then